docs/sources/enterprise/saml/configure-saml.md

```diff
@@ -95,6 +95,22 @@ For Grafana to map the user information, it looks at the individual attributes w
 
 Grafana provides configuration options that let you modify which keys to look at for these values. The data we need to create the user in Grafana is Name, Login handle, and email.
 
+##### The `assertion_attribute_name` option
+
+`assertion_attribute_name` is a special assertion mapping that can either be a simple key, indicating a mapping to a single assertion attribute on the SAML response, or a complex template with variables using the `$__saml{<variable>}` syntax. When Grafana reads the SAML configuration, it uses a regular expression match to determine which form the property is in. Grafana will log helpful error messages on startup if this property is misconfigured.
```

Comment:
```suggestion
`assertion_attribute_name` is a special assertion mapping that can either be a simple key, indicating a mapping to a single assertion attribute on the SAML response, or a complex template with variables using the `$__saml{<attribute>}` syntax. Grafana will log error messages on startup if this property is misconfigured.
```

I’d suggest changing variable to attribute to SAMLify it a little and removing helpful because helpfulness is subjective and might provoke a reader who has no idea what the error message is trying to say. 🙂

I’d also remove the part about regexp, that’s an implementation detail.
I’d suggest adding a sentence about what happens if the assertion is missing or a list.

If I wanted to say set the name field to `The Most Honorable $__saml{surname}`, would that work with this new structure? Asking for a friend 🙃

Reply:
Thanks! I'll clean this up a bit tomorrow after my day off.

Tell your friend that would work just fine!